cupy/cusparse.py

```diff
@@ -89,8 +95,12 @@ def _dtype_to_DataType(dtype):
     'csrgeam2': (9020, None),
     'csrgemm': (8000, 11000),
     'csrgemm2': (8000, None),
+    'csrilu02': (8000, None),
+    'csrilu02_analysis': (8000, None),
+    'csrsv2_solve': (8000, None),
+    'csrsv2_analysis': (8000, None),
     'spmv': (10200, None),
-    'spmm': (10301, None),  # accuracy bugs in cuSparse 10.3.0
```

Comment:
Don't remove comments. (Maybe you were thinking there's no CUDA 10.3? Note that CUDA libraries follow different version semantics than the Toolkit itself.)

Reply:
This was an accident, sorry!